docs/sources/alerting/fundamentals/alert-rules/annotation-label.md

```diff
@@ -24,7 +24,11 @@ weight: 105
 
 # Labels and annotations
 
-Labels and annotations contain information about an alert. Labels are used to differentiate an alert from all other alerts, while annotations are used to add additional information to an existing alert.
```

Comment:
what's wrong with this sentence? For me it offers a quick definition and distinction up front.

Reply:
Labels and annotations contain information about an alert and can be used to optimize and customize your alert rules.

Reply:
nothing wrong actually!
restored that paragraph

Reply:
haha:)